plugins/Live/API.php

```diff
@@ -144,7 +153,7 @@ public function getLastVisitsForVisitor($visitorId, $idSite, $filter_limit = 10,
      * @param bool|string $period Period to restrict to when looking at the logs
      * @param bool|string $date Date to restrict to
      * @param bool|int $segment (optional) Number of visits rows to return
-     * @param bool|int $countVisitorsToFetch (optional) Only return the last X visits. By default the last GET['filter_offset']+GET['filter_limit'] are returned.
+     * @param bool|int $countVisitorsToFetch DEPRECATED (optional) Only return the last X visits. Please use the API paramaeter 'filter_offset' and 'filter_limit' instead.
```

Comment:
Deprecated `$countVisitorsToFetch` it was only there temporarily for a while to fix the previously not correctly working `filter_offset` & `filter_limit`